context/2022/acr-wcag-21-sc.json-ld

```diff
@@ -0,0 +1,336 @@
+  "WCAG21Tech:failures/F69": { "@type": "earl:TestCase" },
+  "WCAG21Tech:failures/F70": { "@type": "earl:TestCase" },
+  "WCAG21Tech:failures/F73": { "@type": "earl:TestCase" },
+  "WCAG21Tech:failures/F74": { "@type": "earl:TestCase" },
+  "WCAG21Tech:failures/F75": { "@type": "earl:TestCase" },
+  "WCAG21Tech:failures/F78": { "@type": "earl:TestCase" },
+	"WCAG21Tech:failures/F79": { "@type": "earl:TestCase" },
+  "WCAG21Tech:failures/F80": { "@type": "earl:TestCase" },
+  "WCAG21Tech:failures/F81": { "@type": "earl:TestCase" },
+  "WCAG21Tech:failures/F82": { "@type": "earl:TestCase" },
+  "WCAG21Tech:failures/F83": { "@type": "earl:TestCase" },
+  "WCAG21Tech:failures/F85": { "@type": "earl:TestCase" },
+  "WCAG21Tech:failures/F86": { "@type": "earl:TestCase" },
+  "WCAG21Tech:failures/F87": { "@type": "earl:TestCase" },
+  "WCAG21Tech:failures/F89": { "@type": "earl:TestCase" },
+  "WCAG21Tech:failures/F90": { "@type": "earl:TestCase" },
+  "WCAG21Tech:failures/F91": { "@type": "earl:TestCase" },
+  "WCAG21Tech:failures/F92": { "@type": "earl:TestCase" },
+  "WCAG21Tech:failures/F93": { "@type": "earl:TestCase" },
+  "WCAG21Tech:failures/F94": { "@type": "earl:TestCase" },
+	"WCAG21Tech:failures/F95": { "@type": "earl:TestCase" },
+	"WCAG21Tech:failures/F96": { "@type": "earl:TestCase" },
+	"WCAG21Tech:failures/F97": { "@type": "earl:TestCase" },
+	"WCAG21Tech:failures/F99": { "@type": "earl:TestCase" },
+	"WCAG21Tech:failures/F100": { "@type": "earl:TestCase" },
+	"WCAG21Tech:failures/F101": { "@type": "earl:TestCase" },
+	"WCAG21Tech:failures/F102": { "@type": "earl:TestCase" },
+	"WCAG21Tech:failures/F103": { "@type": "earl:TestCase" },
+	"WCAG21Tech:failures/F104": { "@type": "earl:TestCase" },
+	"WCAG21Tech:failures/F106": { "@type": "earl:TestCase" },
+	"WCAG21Tech:failures/F107": { "@type": "earl:TestCase" },
+	"WCAG21Tech:failures/F105": { "@type": "earl:TestCase" },
+	"WCAG22Tech:failures/F108": { "@type": "earl:TestCase" },
+	
+
+	"reportId": { "@id":"wb-acr:reportId", "@type": "xsd:positiveInteger" },
+	"assessment": { "@id": "dct:title" },
+
+	"subjectUrl": { "@id": "http://schema.org/url", "@type": "@id" },
+
+	"assertedBy": { "@id": "earl:assertedBy", "@type": "earl:Assertor" },
+	"producedBy": { "@id": "earl:mainAssertor", "@type": "earl:Assertor" },
+	"name": { "@id": "foaf:name" },
+	"page": { "@id": "foaf:page" },
+	"revisedBy": { "@id": "wb-acr:review", "@type": "wb-arc:Revision" },
+
+	"score": { "@id": "wb-acr:score", "@type": "xsd:string" },
+	"results": { "@id": "earl:result", "@type": "earl:TestResult" },
+
+	"earl:test": { "@type": "@id" },
+	"techniques": { "@id": "earl:test", "@type": "@id" },
+	"advisory": { "@id": "earl:test", "@type": "@id" },
+	"outcome": { "@id": "wb-acr:outcome", "@type": "xsd:string" },
+
+	"info": { "@id": "earl:info" },
+	"label": { "@id": "dct:title" },
+	"notes": { "@id": "dcterms:description" },
+	"date": { "@id": "dcterms:date" },
+	
+	"earl:mode": { "@type": "@id" }
```

Comment:
Add alias for the:
* wb-acr:severity
* wb-acr:relevancy

Plus its corresponding status

* wb-acr:critical
* wb-acr:moderate

Reply:
Added alias for both severity and relevancy in recent commit